### ISSUE TITLE
[Quest][Mechanic] Ring of Blood/Amphitheater of Anguish: Kill credit not given if Killing Blow isn't earned.

### ISSUE THREAD

**Issue by Aridam** [dalaranwow/dalaran-wow#5469]
**Current behaviour**: If one doesn't get the Killing Blow for any of the Elites in the Ring of Blood/Amphitheater of Anguish, the Quest credit won't be given to them -- currently trying to do it with other people as such: I tag the Elite and try to kill it with helper A, but since if I don't get the last hit to gain the Killing Blow on the Elite, I don't ever get the Quest credit.

**Expected behaviour**: Killing Blow shouldn't be required if the Elite is tagged by the doer of the Quest already -- tag alone is enough.

**Steps to reproduce the problem**:

1. Go to either the RoB or the AoA;
2. Do any of the Quests in the chains;
3. Don't get the KB on the required Elite after tagging it -- no Quest credit.

**Comment by Aridam** [comment]
Like general Mob tagging -- tag the Mob and then nothing else is needed (that's also how one powerlevels another -- same basis). Basically as long as the Mob is "yours", so long as it dies by any means, credit's yours.

**Comment by TeridanDW** [comment]
> Like general Mob tagging -- tag the Mob and then nothing else is needed (that's also how one powerlevels another -- same basis). Basically as long as the Mob is "yours", so long as it dies by any means, credit's yours.

I agree with this statement (in wrath), creature is locked to the person/group who attacked it first (raid members excluded if the quest is not marked as "raid"). The case mentioned above shouldn't be any diffrent than the general rule.

Note: This rule got changed in Legion where player can get quest credit even if he attacked an already tagged creature with 2 exeptions:
 - **normal** creature has under 10 players in his threat list (elite,rare,boss creatures excluded)
 - creature tagged by player from oposite faction

**Comment by thestranger064** [comment]
Yes, I have done several reports that lists this behaviour, but was closed because someone decided to link to opinions threads - and treat them as facts. A comment in a forum post is not a proof but , it seems it is here.

**Comment by torindel** [comment]
> Like general Mob tagging -- tag the Mob and then nothing else is needed (that's also how one powerlevels another -- same basis). Basically as long as the Mob is "yours", so long as it dies by any means, credit's yours.

Correct: On retail wotlk mobs get "group tagged/tapped":
- whoever damages (including player pets) mob first taggs/tapps it to his group (including 1 person group)
- it's impossible to use quest items on mob that was tapped by another group
- after mob dies (regardles of reasons/damage sources) everyone in group that tapped mob gets kill credit/exp/reputation/loot rights assuming that they are close enough (including people that died but havent released spirit)
- its possible to invite someone to group after mob is tapped (damaged), and that person will get kill credit too as long as he accepts invite before mob dies

As for raid: being in raid prevents getting kill/action quest credits to all non raid quests regardles of anything and prevents droping of non-raid quest items

http://wowwiki.wikia.com/wiki/Patch_0.6
> Now the first player or group to damage a monster will "tap" them. When the tapped monster dies, that player or group will get the experience and loot from the kill. Other players will be able to tell if a monster is tapped because it's portrait will turn grey. Players are free to help kill the tapped monster, but they will receive no experience or loot from the kill. Even with help, the player or group who initially tapped the monster will gain full experience from the kill. In situations where a much higher level player helped with the kill, some of the experience reward will be reduced.

http://wowwiki.wikia.com/wiki/Patch_0.10
> You will now get bounty credit if you die and do not release your spirit. For example, if you are on the Leaders of the Fang Quest and die while fighting a Fang Leader, you will still get credit for the kill if your party kills the leader before you release your spirit.

http://wowwiki.wikia.com/wiki/Patch_1.2.0
> Players will now receive credit for killing a monster even if they die during battle.

http://wowwiki.wikia.com/wiki/Patch_1.2.2
> Hunter and Warlock pets will now correctly "tap" the targets they attack, thus giving the pet owner kill credit for a target even if another player later attacks the same target.

http://wowwiki.wikia.com/wiki/Patch_3.0.8
> Tapping: All player spells which cause a creature to become aggressive to you will now also immediately cause the creature to be tapped.

http://wowwiki.wikia.com/wiki/Tap?oldid=2066004 (October 4, 2009 version)
> Tapped is a state that occurs when a player damages a mob, locking its experience gain, loot and corpse to be available only to that player or anyone in that player's party. Another term for a tap is a "tag".
> This state is indicated by the mob's health bar turning gray.
> Only attacks and direct damage abilities are able to tap a mob; Damage over Time abilities and non-damaging debuffs will not prevent another player from tapping a mob.
> Trying to use an item on a mob that has been tagged by someone else or someone from outside their party will return the error message "Target is tapped." However, players can still attack and kill the tapped mob even if they were not the ones to tap it.
> It is possible for one player to tag multiple mobs in order to keep others from doing so. Although controversial, it is not against Blizzard's Rules of Conduct to tag multiple mobs.
